apps/ios/SimpleXChat/ChatTypes.swift

```diff
@@ -2099,6 +2099,48 @@ public struct ChatItem: Identifiable, Decodable {
         }
     }
 
+    public var mergeCategory: CIMergeCategory? {
+        switch content {
+        case .rcvChatFeature: .chatFeature
+        case .sndChatFeature: .chatFeature
+        case .rcvGroupFeature: .chatFeature
+        case .sndGroupFeature: .chatFeature
+        case .rcvGroupEvent(.memberCreatedContact): nil
+        case .rcvGroupEvent: .rcvGroupEvent
```

Comment:
this may be a bit too aggressive - probably it shouldn't merge role changes (especially of user), member removal